### PR TITLE
Refactor to remove configuration files and use one deploy

### PR DESCRIPTION
**Pull request by ecwillis**
## Changes in this PR.
- Adds a `node-lambda test` function to test lambda functions with (per #6)
- Refactors `create, update, deploy` to single method `deploy` (per #8 )

### CODE REVIEW

File: lib/bundler.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ const browserify = require('browserify');
 const babelify = require('babelify');
 
 const bes2015 = require('babel-preset-es2015');
+const protoAssign = require('babel-plugin-transform-object-set-prototype-of-to-assign');
```
Comment (jgallen23) [inline lib/bundler.js:10]: I think there is a babel polyfill that apex is using that might fix all of the issues

File: lib/configure.js [hunk 2]
```diff
@@ -10,25 +10,36 @@ const dotenv = require('dotenv');
...
+  async.auto({
+    buildConfig: (cb, results) => {
+      
+      let configObj = _.reduce(confFiles, (map, file) => {
         let config = dotenv.parse( fs.readFileSync(file));
         map = _.defaults({}, map, config);
         return map;
       }, {});
+      
+      console.log(configObj);
```
Comment (jgallen23) [inline lib/configure.js]: lets remove console

**Comment by jgallen23** [comment]
so biggest issue right now is it copies things to the temp directory and the runs browserify.  This won't work when calling a function in the node_modules folder.  All of the dependencies come back undefined.  We need to run browserify against the handler file in it's directory and then save the output to the temp directory.

**Comment by ecwillis** [comment]
The latest commit above swaps out the bundle and configure tasks, and runs the bundle task from where the file lives (and not from the temp dir).

**Pull request merged**